src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -17,6 +17,9 @@
     <TargetFramework>net5.0</TargetFramework>
     <Prefer32Bit>false</Prefer32Bit>
 
+    <IsPublishable>true</IsPublishable>
+    <RuntimeIdentifiers>win-x64;linux-x64;osx-x64</RuntimeIdentifiers>
```

Comment:
><RuntimeIdentifiers>win-x64;linux-x64;osx-x64</RuntimeIdentifiers> [](http://example.com/codeflow?start=4&length=66)

Do we need this? It would be nice to avoid needing to put this in multiple places.